docs/serverless/08-04-trigger-function-with-event.md

```diff
@@ -105,24 +105,52 @@ To test if the Trigger CR is properly connected to the function:
     }
     ```
 
-2. Send an event manually to trigger the function:
+2. Send an event manually to trigger the function. The event payload is delivered as the body of the HTTP request in `JSON` format. You can use [CloudEvents](https://github.com/cloudevents/spec/blob/master/spec.md) specification or the previous, Kyma-compatible solution.
+
+   <div tabs name="examples" group="test=trigger">
+    <details>
+    <summary label="CloudEvents">
+    CloudEvents-compatible event payload
+    </summary>
 
     ```bash
-    curl -X POST -H "Content-Type: application/json" https://gateway.{CLUSTER_DOMAIN}/$APP_NAME/v1/events -k --cert {CERT_FILE_NAME}.crt --key {KEY_FILE_NAME}.key -d \
-    '{
-        "event-type": "{EVENT_TYPE}",
-        "event-type-version": "{EVENT_VERSION}",
-        "event-time": "2020-04-02T21:37:00Z",
-        "data": "123456789"
-    }'
+        curl -v -k --cert {CERT_FILE_NAME} --key {KEY_FILE_NAME} -d '{
```

Comment:
Is there a reason we wanna have this indention in the rendered document?

Reply:
No, fixed it :)